rules/windows/discovery_system_time_discovery.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2023/01/17"
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/01/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the usage of commonly used system time discovery techniques, which attackers may use during the reconnaissance phase after compromising a system. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
```

Comment:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
```

Reply:
@Aegrah, I forgot these. We need `logs-windows.*` for sysmon compatibility and endgame for endgame EDR compatibility